app/assets/stylesheets/application.scss

```diff
@@ -173,3 +173,33 @@ table {
     line-height: 1.25;
   }
 }
+
+.banner-dangerous {
```

Comment:
Dunno if dangerous is a good word. Maybe ramification or significance or connotation, or something like that.